apps/financial-aid/web-osk/src/utils/useFileUpload.ts

```diff
@@ -71,7 +76,7 @@ export const useFileUpload = (formFiles: UploadFile[]) => {
   const createSignedUrl = async (
     filename: string,
   ): Promise<SignedUrl | undefined> => {
-    const validFileName = filename.replace(/ +/g, '_')
+    const validFileName = filename ? filename.replace(/ +/g, '_') : ''
```

Comment:
Why are we adding the ternarny here?

Reply:
If file name is undefined, it was on refresh when fileneame was empty, I added this just in case, should I remove it since it should happen, or have this still here just in case?

Reply:
Yes like we talked about IRL then this should not really happen and I think we probably don't want an unnamed file 😄